src/components/_overlays/Dialog/Dialog.styles.ts

```diff
@@ -114,10 +114,9 @@ export const FooterButtonsContainer = styled.div`
   ${media.sm} {
     grid-auto-flow: column;
     grid-auto-columns: auto;
-    justify-content: end;
 
-    > :first-of-type {
-      order: 2;
-    }
+    /* those two properties below will reverse order of button */
+    direction: rtl;
```

Comment:
`direction` has a quite specific meaning and it supposed to be used only for different languages that use` rtl`. Let's find something else for styling